### DISCUSSION TITLE
Support a bigger Display

### DISCUSSION THREAD

**Discussion by raindancer2204** [steadramon/ESPGeiger#17] [Ideas]
Hi,

at the moment only the follwooing Display is suppored "SSD1306". Is there a possiblity to ad also a bigger Display? For example the "SSD1309". Its better to see it without glases :)

**Comment by steadramon** [comment]
Have you tried a I2C SSD1309? There's a possibility that it could work

**Comment by raindancer2204** [comment]
Not at the moment i have to order one and test it if you say it could work..

**Reply by steadramon** [reply]
I'm mostly going from reading other reports from the OLED display library. I'm not clear/sure it works - if you had one already it would be good to test.

**Comment by steadramon** [comment]
I've tested this with a "SSD1309" like the following -

![image](https://github.com/user-attachments/assets/a00df215-b174-408b-b624-e4c92ab044b1)

Seems to be working fine out of the box (esp8266_oled build) :) 

![image](https://github.com/user-attachments/assets/b5d726e9-ca75-4aee-a459-c14a444ac04c)

Note this is the [I2C 2.4 inch version](https://s.click.aliexpress.com/e/_Ddrd945) (black board), the 1.3 inch version I tried didn't work

**Comment by steadramon** [comment]
Support for SSD1306 / SH1106 / SSD1309 have all been integrated since version 0.11.0 - this brings a number of new options to the table in terms of displays.
